ShovelModeAIDriver.lua

```diff
@@ -66,13 +66,19 @@ function ShovelModeAIDriver:init(vehicle)
 	self.shovelState = self.states.STATE_TRANSPORT
 	self.refSpeed = 15
 	self:setHudContent()
+	self.foundTrailer = nil
 end
 
 function ShovelModeAIDriver:setHudContent()
 	courseplay.hud:setShovelModeAIDriverContent(self.vehicle)
 end
 
 function ShovelModeAIDriver:start()
+	if not self:findShovel(self.vehicle) then 
+		print("Error: shovel not found!!")
```

Comment:
There's and AIDriver:error() similar to debug() so you could have used self:error() here, you may want to change it in the next push.